lib/template.html

```diff
@@ -5,21 +5,13 @@
 {% endif %}
 
 {% assign seo_site_title = site.title | default: site.name %}
+{% assign seo_page_title = page.title | default: seo_site_title %}
+{% assign seo_title = page.title | default: seo_site_title %}
 
-{% if page.title %}
-  {% assign seo_title = page.title %}
-  {% assign seo_page_title = page.title %}
-
-  {% if seo_site_title %}
-    {% assign seo_title = seo_title | append:" - " | append: seo_site_title %}
-  {% endif %}
-{% elsif seo_site_title %}
-  {% assign seo_title = seo_site_title %}
-  {% assign seo_page_title = seo_site_title %}
-
-  {% if site.description %}
-    {% assign seo_title = seo_title | append:" - " | append: site.description %}
-  {% endif %}
+{% if page.title and seo_site_title %}
+  {% assign seo_title = seo_title | append:" | " | append: seo_site_title %}
+{% elsif site.description %}
+  {% assign seo_title = seo_title | append:" | " | append: site.description %}
```

Comment:
That does not appear to be what L70 is testing.

I am talking about the case where `page.title` is set, `site.name` is not set, `site.description` is set.

In this case, the first `if` condition is false because there is no `seo_site_title`, but the `elsif` is true because `site.description` does exist.

So our title becomes `page.title | site.description` :-1: